options/locale/locale_en-US.ini

```diff
@@ -259,7 +259,7 @@ openid_signin_desc = Enter your OpenID URI. For example: https://anne.me, bob.op
 disable_forgot_password_mail = Account recovery is disabled. Please contact your site administrator.
 email_domain_blacklisted = You cannot register with your email address.
 authorize_application = Authorize Application
-authroize_redirect_notice = You will be redirected to %s if you authorize this application.
```

Comment:
How does this affect Crowdin?

Reply:
They'll have to be redone unfortunately but I thought it worth fixing.

Reply:
@zeripath If this gets backported, would could you apply the changes to the other translation files too? I think this is a reasonable PR to backport, but I'll leave that up to you if you have time.

Reply:
I can drop it if preferred